apps/api/pkg/handlers/tenant.go

```diff
@@ -32,28 +32,41 @@ func TenantHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func createTenantHandler(w http.ResponseWriter, r *http.Request) {
-	if r.Method != http.MethodPost {
-		http.Error(w, "Invalid request method", http.StatusMethodNotAllowed)
-		return
+	type Payload struct {
+		OrganizationName string `json:"organization_name"`
+		CreatorId        string `json:"creator_id"`
```

Comment:
Why do you need to get CreatorId in body? You have an Id in token. You can use that id.